doc/source/tutorials/basictutorial.rst

```diff
@@ -0,0 +1,656 @@
+	// print "hello world 1".
+	print "hello world 2".
+
+If you'd run these two lines of code you'd **ONLY** see:
+``hello world 2``
+
+You'd might be wondering why you'd want to use the forward slashes if the script doesn't read them.
+The computer ignores them, but they're there to help leave explanations for the humans to see who might have to read your program.
+
+To clear the screen type: ::
+
+	clearscreen.
+
+Now say that we really like to use our hello world command but don't want to type
+the entire sentence every time, we could use the set command. ::
+
+	set x to "hello world".
+
+The set command 'sets' a certain value to the given variable. Everytime we refer
+to ``x`` we will actually refer to ``"hello world"``. ::
+
+	set x to "hello world".
+	print x. // shows: hello world
+
+Of course we can choose other things to print other than ``hello world``.
+Keep in mind that 'normal' text requires ``""`` and variables you made using the
+set command, numbers and booleans (true or false) don't need ``""``. ::
+
+	set x to "hello world".
+	set y to true.
+	set z to 123.
+
+	print x.   // shows: hello world
+	print "x". // shows: x
+	print y.   // shows: true
+	print "y". // shows: y
+	print z.   // shows: 123
+	print "z". // shows: z
+
+You can also replace a variable you've made: ::
+
+	set x to "hello world".
+	set x to "updated text".
+	print x. // shows: updated text
+
+	set x to "hello world".
+	print x. // shows: hello world
+	set x to "updated text".
+	print x. // shows: updated text
+
+As you can see ``hello world`` doesn't exist anymore. If you'd want to print
+both you could do: ::
+
+	set x to "hello world".
+	set y to x.
+	set x to "updated text".
+
+	print y. // shows: hello world
+	print x. // shows: updated text
+
+Variables don't just have to be one letter you could also use a word as a variable, don't use spaces when naming variables. ::
+
+	set WhateverThisVariableIs to false.
+	print WhateverThisVariableIs. // shows: false
+
+=============
+If statements
+=============
+
+Now we know how to set certain text to a variable we can explore more stuff.
+For instance ``if``, this checks if a certain value matches the given value. ::
+
+  set x to 1.
+
+  if x = 1 {
+    print "x is one".
+  }
+
+This will show: ``x is one``.
+
+You've probably noticed the curly brackets ``{ }`` after an ``if`` statement. You don't need a period at the end of an ``if`` statement but you need these brackets.
+You could cover the piece the code within the curly brackets with your hand and say: if ``x`` is equal to ``1``, then do whatever
+I covered with my hand. ``If`` statements can also be used for booleans: ::
+
+  set SomeBoolean to true.
+
+  if SomeBoolean = true {
+    print "this is a true".
+  }
+
+This will show: ``this is true``.
+
+Ofcourse the equals sign isn't the only symbol you can use, other symbols are:
+
+
+Equals to or bigger than:
+
+1 >= 1
+
+2 >= 1
+
+Equals to or smaller than
+
+1 <= 1
+
+1 <= 2
+
+Is not equal to:
+
+1 <> 2
```

Comment:
REST markdown likes to do silly things with '<' and '>' sometimes.  It's probably best to protect the above examples with double-backticks.